lib/hacluster.pm

```diff
@@ -92,6 +93,15 @@ sub get_node_to_join {
     return get_required_var('HA_CLUSTER_JOIN');
 }
 
+sub get_ip {
+    my $node_hostname = shift;
+    $_ = script_output "host -t A $node_hostname";
```

Comment:
Better to not use `$_` as the variable name as it's a special variable name in perl. You can safely change it for something else in this line and the next one.